src/units/unit.hpp

```diff
@@ -94,6 +94,7 @@ class unit_ability_list
 	const unit_ability& front() const  { return cfgs_.front(); }
 	unit_ability&       back()         { return cfgs_.back();  }
 	const unit_ability& back()  const  { return cfgs_.back();  }
+	int                 size()        { return cfgs_.size(); }
```

Comment:
A vector's size() returns an std::size_t, not an int.

Reply:
fixed